src/rest-server/docs/swagger.yaml

```diff
@@ -815,29 +892,338 @@ components:
           type: string
       required:
         - message
-    JobConfig:
-      type: object
-      description: TODO
-      properties: {}
     JobProtocol:
       type: object
-      description: TODO
-    JobSummary:
-      type: object
+      description: openpai job protocol
       properties:
+        protocolVersion:
```

Comment:
how to make sure this definition is synchronzied with openpai-protocol repo?

Reply:
there's no such way currently, maybe we can use [`$ref`](https://swagger.io/docs/specification/using-ref/) after we migrate protocol repo to openapi spec

Reply:
yes. and I think openapi spec is clear and strong enough to describe the protocol